### PR TITLE
docs: edit two-way binding doc

### PR DESCRIPTION
**Pull request by kapunahelewong**
This commit edits copy and doc regions to clarify concepts.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Edit of two-way binding doc copy and docregions.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: aio/content/guide/two-way-binding.md [hunk 1]
```diff
@@ -1,76 +1,77 @@
-# Two-way binding `[(...)]`
+# Two-way binding
 
-Two-way binding gives your app a way to share data between a component class and
-its template.
+Two-way binding gives components in your application a way to share data.
+Use two-way binding binding to listen for events and update values simultaneously.
```
Comment (aikidave) [inline aio/content/guide/two-way-binding.md:4]:
```suggestion
Use two-way binding binding to listen for events and update values simultaneously between parent and child components.
```

File: aio/content/guide/two-way-binding.md [hunk 2]
```diff
@@ -1,76 +1,77 @@
...
+The `[()]` syntax combines the brackets of property binding, `[]`, with the parentheses of event binding, `()`, as follows.
 
-</div>
+<code-example path="two-way-binding/src/app/app.component.html" header="src/app/app.component.html" region="two-way-syntax"></code-example>
+
+## How two-way binding works
+
+The following `sizerComponent` has a `size` value property and a `sizeChange` event.
+The `size` property is an `@Input()`, so data can flow into the `sizerComponent`.
+The `sizeChange` event is an `@Output()`, which allows data to flow out of the `sizerComponent` to the parent component.
```
Comment (aikidave) [inline aio/content/guide/two-way-binding.md:37]: I would add an important note or something here:

For two-way databinding to work you, the `@Output()` property must use the pattern, `inputChange`, where `input` is the name of the input property. For example, if the input property is `size`, the output property must be `sizeChange`.

File: aio/content/guide/two-way-binding.md [hunk 3]
```diff
@@ -1,76 +1,77 @@
...
+## Syntax `[()]`
 
-Visualize a *banana in a box* to remember that the parentheses go _inside_ the brackets.
+Angular's two-way binding syntax is a combination of square brackets and parentheses, `[()]`.
+The `[()]` syntax combines the brackets of property binding, `[]`, with the parentheses of event binding, `()`, as follows.
 
-</div>
+<code-example path="two-way-binding/src/app/app.component.html" header="src/app/app.component.html" region="two-way-syntax"></code-example>
+
+## How two-way binding works
```
[the remark this answers is not in the document]
Comment (kapunahelewong) [reply aio/content/guide/two-way-binding.md:33]: This was my thought as well. Should we move it into the conceptual reference area? I believe we also spoke at one point about possibly archiving this one or folding it into another document?

[the remark this answers is not in the document]
Comment (kapunahelewong) [reply aio/content/guide/two-way-binding.md:34]: Note from meeting conversation. We'll iterate in further PRs.

**Review by aikithoughts — APPROVED** [review]
LGTM!

Reviewed-for: global-docs-approvers

**Pull request closed without merging**